tutorials/networking/websocket.rst

```diff
@@ -44,13 +44,13 @@ This example will show you how to create a WebSocket connection to a remote serv
 
     func _ready():
         # Connect base signals to get notified of connection open, close, and errors.
-        _client.connect("connection_closed", self, "_closed")
-        _client.connect("connection_error", self, "_closed")
-        _client.connect("connection_established", self, "_connected")
+        _client.connect("connection_closed", self._closed)
```

Comment:
```suggestion
        _client.connection_closed.connect(self._closed)
```
You can avoid strings entirely in GD4 :)

Reply:
right you are, even better